joomla/joomla-cms
# [4.0] add fa Regular icons back into core availability

Pull Request for Issue # .

### Summary of Changes
adds back the ability to use fontawesome regular fonts into core.


### Testing Instructions
go to /administrator/index.php
inspect eye.
do the same for front end login
apply pr
using the inspector, change "fas fa-eye fa-fw" to "far fa-eye fa-fw"
notice the icon changed.


### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/96077347-29316e80-0e75-11eb-919a-3a628726ba76.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/96077370-3b131180-0e75-11eb-9720-0178f0adcfa0.png)



### Documentation Changes Required
Our implementation of "Fontawesome Regular icons" is a limited 151 icon set from the 1852 icons included in their "pro" version.  This is by design from FA.  They don't mention that ANY regular icons are included in the free set at all on their site.  Only the icons listed [here](https://fontawesome.com/icons?d=gallery&s=regular&m=free) are included.
